Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,501 @@
+nozzle will be optimized for ambient conditions.
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+  % At the end of the day, whether the project ``succeeds'' or ``fails'' is judged against the objectives it sought to meet.
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+The primary objective of this project is to create a cold gas thruster for on-ground experimentation. The goal for the data is to be able to design a nozzle for a small propulsion system, where boundary
+layer effects are dominant, but also to achieve an specific impulse of 50s. Using the thrust over time curves, the specific impulse can be
+calculated numerically using MATLAB\@. The data will be transferred and analyzed there using the following equations as a basis\cite{SPAD}.
+
+
+\begin{equation} \label{total-impulse}
+  I_t    = \int_{t_i}^{t_f} F_{thrust} dt
+\end{equation}
+
+\begin{figure}[ht]
+  \includegraphics[width=\linewidth]{figs/cold-gas-thruster.png}
+  \caption{The Solidworks Assembly of the Spring 2017 design of the SPEX cold gas thruster. The piping from the solenoid to the nozzle is omitted for clarity.}
+\label{fig:lifecycle}
+\end{figure}
+
+\begin{equation} \label{specific-impulse}
+  I_s = \frac{I_t}{m_p g_0}
+\end{equation}
+
+Data error found from past experimentation was 5\% of the thrust. This error is due to the quality of the load cell. It has an inherent inaccuracy. This
+amount of error is acceptable. The performance analysis will not be detrimentally affected. The risk mitigation includes zeroing the load cell just before the test.
+This will erase the effects caused by initial weight placed on the load cell. This initial weight comes from the aluminum sliding fixture which holds the nozzle
+and the plumbing system.
+
+
+
+% \section{Secondary Objectives}
+% \label{sec:secondary-obj}
+% Secondary Objectives are lower priority or bonus objectives that are significant but not the main focus of the project. This template does not have secondary objectives.
+
+\section{Benefit to SPEX}
+\label{sec:benefit}
+% One of the core values of SPEX is to provide opportunities for academic and professional growth for its members,
+% and to challenge them with interesting projects.
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+
+This project can be benefical to SPEX because of the experience given to SPEX in propulsion system design and experimentation. The experience includes learning how to contruct flow systems, design nozzles
```

Comment:
Reword for clarity

>This project can be beneficial to SPEX because

is redundant since the section header is `Benefit to SPEX`.  Start with a strong statement and provide supporting evidence in following sentences.